Source/Extensions/Blazorise.Charts/BaseChartPlugin.cs

```diff
@@ -0,0 +1,102 @@
+using System;
+using System.Threading.Tasks;
+using Blazorise.Extensions;
+using Blazorise.Modules;
+using Microsoft.AspNetCore.Components;
+using Microsoft.JSInterop;
+
+namespace Blazorise.Charts;
+
+public abstract class BaseChartPlugin : BaseComponent, IAsyncDisposable
```

Comment:
Should we name this simple as ChartPlugin? Drop the Base?

Reply:
Yes. That's right.